third_party/proto/panacea/datadeal/v2/deal.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 package panacea.datadeal.v2;
 
-option go_package = "github.com/medibloc/panacea-core/x/datadeal/types";
+option go_package = "github.com/medibloc/panacea-core/v2/x/datadeal/types";
```

Comment:
I didn't update this manually. I just copied this file from https://github.com/medibloc/panacea-core/pull/627 :)